### ISSUE TITLE
var defined to undefined...

### ISSUE THREAD

**Issue by Erikkuchan** [BonsaiDen/JavaScript-Garden#151]
[Quote]
foo; // 'undefined'
foo(); // this raises a TypeError
var foo = function() {};

Due to the fact that var is a declaration that hoists the variable name foo before the actual execution of the code starts, foo is already defined when the script gets executed.

But since assignments only happen at runtime, the value of foo will default to undefined before the corresponding code is executed.
[/Quote]

I think this part is confusing.
You are saying "foo is already defined" just after the code snippet saying: foo; // 'undefined'
And then " foo will default to undefined"..
So in short, that makes "foo is defined to undefined.."
Maybe you should say (but I am not sure that is the right terminology for Javascript) that "foo is already DECLARED".

The same comment is true for the Japanese version.
